shiny/bookmark/_button.py

```diff
@@ -28,6 +28,30 @@ def input_bookmark_button(
 
     A `bookmarkButton` is a [input_action_button()] with a default label that consists of a link icon and the text "Bookmark...". It is meant to be used for bookmarking state.
 
+    Default behavior
```

Comment:
Also feels like this should come after `Parameters`?

```suggestion
    Multiple (module) buttons
```